File: ios/maimaid/Models/UtageNoteTypes.swift
struct UtageNoteTypes: Decodable, Sendable {
    let tap: Int
    let hold: Int
    let slide: Int
    let touch: Int
    let breakCount: Int

    enum CodingKeys: String, CodingKey {
        case tap, hold, slide, touch
        case breakCount = "break"
    }
}

File: android/app/src/main/java/org/rhythmeta/maimaid/core/data/UtageNoteTypes.kt
package org.rhythmeta.maimaid.core.data

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class UtageNoteTypes(
    val tap: Int,
    val hold: Int,
    val slide: Int,
    val touch: Int,
    @SerialName("break") val breakCount: Int,
)
